### PR TITLE
Modified the highlighted section

### PR DESCRIPTION
**Pull request by Coder-Srinivas**
Modfied the highlighted section based on the [ticket](https://openstax.atlassian.net/browse/RENG-107?atlOrigin=eyJpIjoiYzg2MzkyNjJjZGQ0NGNiYzg3NTFjNzVlM2UyY2ZlYTgiLCJwIjoiaiJ9)

[Design Spec](https://www.figma.com/design/MVxS1h1qovYyDqbWf3DTyF/Version-3.0-Apr24?node-id=1706-1545&m=dev)

Changes made:

- Added an option to view all studies which focus on all the studies when clicked.
- Added a section title and secondary section text as per the design spec

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "@emotion/css": "^11.10.0",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.10.0",
+    "@iconify-icons/ep": "^1.2.12",
```
Comment (chrisbendel) [inline frontend/package.json:26]: lets try to stick to `tabler/icons-react` or decide on a single set of icons to use in general, i think we have too many in the project now 😆

File: frontend/src/screens/learner.tsx [hunk 2]
```diff
@@ -21,23 +21,51 @@ const HighlightedStudies: FC = () => {
...
+                        <CuratedStudies />
+                        {isMobile ?
+                            <MobileStudyCards studies={highlightedStudies} /> :
+                            <DesktopStudyCards studies={highlightedStudies} />
+                        }
+                    </Flex>
+                    <Flex c={colors.green} justify='center' align='center'>
+                        <Flex direction='column' justify='center' align='center' style={{ cursor: 'pointer' }} onClick={()=> handleClick()}>
+                            <Text size='sm'>View all studies</Text>
+                            <Icon icon='arrowDown' width='.875rem'></Icon>
```
Comment (chrisbendel) [inline frontend/src/screens/learner.tsx:45]: prefer to use `tabler` here

File: frontend/src/screens/learner.tsx [hunk 3]
```diff
@@ -21,23 +21,51 @@ const HighlightedStudies: FC = () => {
     const { highlightedStudies } = useParticipantStudies()
     const isMobile = useIsMobileDevice()
 
+    const handleClick = () => {
```
Comment (chrisbendel) [inline frontend/src/screens/learner.tsx:24]: Maybe something a bit more descriptive than the generic `handleClick` - something like `scrollToStudies` or something?

File: frontend/src/screens/learner.tsx [hunk 4]
```diff
@@ -21,23 +21,51 @@ const HighlightedStudies: FC = () => {
...
+                </Flex>
             </Container>
         </Box>
     )
 }
 
+const CuratedStudies: FC = () => {
+
+    return (
+        <Flex c={colors.white} direction='column' w='280px' h='350px' pt='2.25rem' mt='1rem'>
```
Comment (chrisbendel) [inline frontend/src/screens/learner.tsx:57]: Just make sure this container looks okay on mobile

Additionally, remember to try and use <Stack> or <Group> where possible as they are a bit easier to see at a glance what's going on

File: frontend/src/screens/learner.tsx [hunk 5]
```diff
@@ -21,23 +21,51 @@ const HighlightedStudies: FC = () => {
...
         <Box bg={colors.navy} py='md'>
             <Container>
-                <Stack>
-                    <Title c='white' order={2}>Highlighted Studies</Title>
-                    {isMobile ?
-                        <MobileStudyCards studies={highlightedStudies} /> :
-                        <DesktopStudyCards studies={highlightedStudies} />
-                    }
-                </Stack>
+                <Flex direction='column'>
```
Comment (chrisbendel) [inline frontend/src/screens/learner.tsx:34]: this whole section can be Flex -> Group/Stack

File: frontend/src/screens/learner.tsx (hunk 3 above)
Comment (Coder-Srinivas) [reply frontend/src/screens/learner.tsx:24]: Okay

File: frontend/src/screens/learner.tsx (hunk 4 above)
Comment (Coder-Srinivas) [reply frontend/src/screens/learner.tsx]: Okay

File: frontend/src/screens/learner.tsx (hunk 5 above)
Comment (Coder-Srinivas) [reply frontend/src/screens/learner.tsx]: Modified

**Review by chrisbendel — APPROVED** [review]
Looks good

**Pull request merged**